#164139760 Setup postgres and sequelize ORM

danieldare opened the pull request:
#### What does this PR do?
- Set up Postgres Database and Sequelize ORM
- Refactors code-base to remove unused code.
#### Description of Task to be completed?
- create a .sequelizerc file.
- create the model, migration, seeder and config folder using Sequelize CLI.
#### How should this be manually tested?
- run `npm install` to install all dependencies
- run `npm start` to start the application
#### Any background context you want to provide?
This is required to set up the Postgres database on starting the application.
#### What are the relevant pivotal tracker stories?
[#164139760](https://www.pivotaltracker.com/story/show/164139760)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

danieldare committed:
chore(postgres): setup postgres and sequelize ORM
- setup Sequelize ORM
- create user model
- refactor codebase to clear unused files

[Finishes #164139760]

danieldare committed:
chore(postgres): setup postgres and sequelize ORM
- setup Sequelize ORM
- create user model
- refactor codebase to clear unused files

[Finishes #164139760]

LordUche reviewed (commented):
- I observed that the database password is visible in the `config.json` file. It should be defined in a `.env` file.

danieldare committed:
chore(postgres): setup postgres and sequelize ORM
- setup Sequelize ORM
- create user model
- refactor codebase to clear unused files

[Finishes #164139760]

danieldare committed:
Merge branch 'develop' into ch-setup-postgres-164139760
